submit_and_compare/views.py

```diff
@@ -0,0 +1,303 @@
+"""
+Handle view logic for the XBlock
+"""
+from __future__ import absolute_import
+import logging
+
+from django.utils.translation import ungettext
+from lxml import etree
+from six import StringIO
+from xblock.core import XBlock
+from xblockutils.resources import ResourceLoader
+
+from .mixins.fragment import XBlockFragmentBuilderMixin
+
+
+LOG = logging.getLogger(__name__)
+
+
+def _convert_to_int(value_string):
+    """
+    Convert a string to integer
+
+    Default to 0
+    """
+    try:
+        value = int(value_string)
+    except ValueError:
+        value = 0
+    return value
```

Comment:
Can this not get moved out to a utilities file or something?

Reply:
I think we typically don't split out helpers if they're only used in one place (this is marked as a private method, not public).

As mentioned previously, I'll eventually be creating `stanford-xblock-utils` library and if this is used by other things, we can consider moving it then.